app/src/main/kotlin/com/wire/android/ui/authentication/create/common/CreateAccountBaseViewModel.kt

```diff
@@ -291,6 +294,12 @@ private fun RegisterClientResult.Failure.toCodeError() = when (this) {
     RegisterClientResult.Failure.TooManyClients -> CreateAccountCodeViewState.CodeError.TooManyDevicesError
     RegisterClientResult.Failure.InvalidCredentials -> CreateAccountCodeViewState.CodeError.DialogError.InvalidEmailError
     is RegisterClientResult.Failure.Generic -> CreateAccountCodeViewState.CodeError.DialogError.GenericError(this.genericFailure)
+    RegisterClientResult.Failure.PasswordAuthRequired -> {
+        Log.wtf("RegisterClient", "wrong password when register client after creating a new account")
```

Comment:
I just noticed that we don't have wtf log in our kalium logger  which is a (Assert), so maybe can add todo to replace this once we create that function